test/e2e/terraform/packer/consul-ami/scripts/telemetry.json

```diff
@@ -0,0 +1,7 @@
+{
+"datacenter": "west-new",
```

Comment:
Should this be templated with the `aws_region` instead?

Reply:
This sets the Consul datacenter, so.. in short, it can be? But I was gonna add in the README some examples of tags users could put and then let people decide? 
What do you think?

Reply:
Well this would be how it shows up in Datadog right? So what if I spin up things in us-east-1 and us-east-2? Then `west-new` doesn't seem to make sense.